commands/christmaspatrol.js

```diff
@@ -0,0 +1,22 @@
+module.exports = function (bb8,options) {
+ var radius = options.radius; //mm
+ var distance = 2*Math.PI*radius;
+ var colors = ["red","green"];
+ var i = 0;
+ return setInterval(function() {
+    var direction = 30;
```

Comment:
direction can be pulled out of the setInterval so it doesn't get redeclared every 500ms